### PR TITLE
[tool update] Trimal

### PR DESCRIPTION
**Pull request by shiltemann**
Update wrapper for [Trimal](https://trimal.readthedocs.io/en/latest/index.html), a tool for trimming multiple alignment files.

Added some more parameters and small tweaks 

Part of a larger effort to create a gene family tree pipeline 

FOR CONTRIBUTOR:
* [x] I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] License permits unrestricted use (educational + commercial)
* [ ] This PR adds a new tool or tool collection
* [x] This PR updates an existing tool or tool collection
* [ ] This PR does something else (explain below)

### CODE REVIEW

**Comment by shiltemann** [comment]
oh! guess I just missed https://github.com/galaxyproject/tools-iuc/pull/6397 by @B0r1sD

**Comment by bgruening** [comment]
Upps, sorry. its available here: https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/trimal/trimal/1.5+galaxy0

**Comment by shiltemann** [comment]
cool, I will rebase and see if there is anything I have in my version that can enhance the current wrapper or not

File: tools/trimal/trimal.xml [hunk 1]
```diff
@@ -31,21 +85,44 @@
...
             <when value="-noallgaps"/>
             <when value="-gappyout"/>
             <when value="-strict"/>
             <when value="-strictplus"/>
             <when value="-automated1"/>
+            <when value="manual">
+                <param argument="-selectcols" type="text" optional="true" label="Columns to be removed from alignment" help="Example: 2,4,8-12 Range: [0 - (Number of Columns - 1)]">
+                    <validator type="regex" message="must be comma-separated list of whole numbers">(\d+)([,-]\d+)*</validator>
+                </param>
+                <param argument="-selectseqs" type="text" optional="true" label="Sequences to be removed from alignment" help="Example: 2,4,8-12 Range: [0 - (Number of Sequences - 1)]">
```
Comment (bgruening) [inline tools/trimal/trimal.xml:101]:
```suggestion
                <param argument="-selectseqs" type="text" optional="true" label="Sequences to be removed from alignment" help="Example: "2,4,8-12"; Range: [0 - (Number of Sequences - 1)]">
```

Maybe something like that, to separate it more?

File: tools/trimal/trimal.xml [hunk 2]
```diff
@@ -31,21 +85,44 @@
...
+                <option value="manual">manual mode - specify columns or sequences to remove</option>
             </param>
             <when value="-nogaps" />
             <when value="-noallgaps"/>
             <when value="-gappyout"/>
             <when value="-strict"/>
             <when value="-strictplus"/>
             <when value="-automated1"/>
+            <when value="manual">
+                <param argument="-selectcols" type="text" optional="true" label="Columns to be removed from alignment" help="Example: 2,4,8-12 Range: [0 - (Number of Columns - 1)]">
```
Comment (shiltemann) [inline tools/trimal/trimal.xml:98]:
```suggestion
                <param argument="-selectcols" type="text" optional="true" label="Columns to be removed from alignment" help="Example: "2,4,8-12"; Range: [0 - (Number of Columns - 1)]">
```

**Review by bgruening — APPROVED** [review]

**Comment by shiltemann** [comment]
Thanks for the fixes @bgruening!

**Comment by B0r1sD** [comment]
Seems like enhancements were definitely made, thanks @shiltemann!
Ping @nicodr97 and @pauldg for visibility.

**Comment by nicodr97** [comment]
Wonderful! Would it make sense to remove the html output by default? Or at least the possibility for the user not to generate it? Thank you for the contributions :)

**Comment by shiltemann** [comment]
@nicodr97 sure, we can make that choice for users if you think that is useful :+1:

**Pull request merged**